aspnetcore/razor-pages/sdk.md

```diff
@@ -100,3 +100,13 @@ The Razor SDK defines two primary targets:
 * By default, the Razor SDK doesn't publish reference assemblies that are required to perform runtime compilation. This results in compilation failures when the application model relies on runtime compilation&mdash;for example, the app uses embedded views or changes views after the app is published. Set `CopyRefAssembliesToPublishDirectory` to `true` to continue publishing reference assemblies.
 
 * For a web app, ensure your app is targeting the `Microsoft.NET.Sdk.Web` SDK.
+
+## Razor language version
+
+The Razor parser and code generator version is controlled by the `<RazorLangVersion>` MSBuild property. In rare cases, an app requires a different Razor language version than the version of the shared framework used by the app. The version is set in the app's project file:
```

Comment:
When targeting the `Microsoft.NET.Sdk.Web` SDK, `RazorLangVersion` is inferred from the the app's target framework version. For projects targeting the `Microsoft.NET.Sdk.Razor` SDK or in the rare cases that the app requires requires a different Razor language version than the inferred value, a version can be configured by setting the property in the application's project file: